redesign/cs-smg.html

```diff
@@ -0,0 +1,204 @@
+<!DOCTYPE html>
```

Comment:
We need to rethink these routes. Whilst we don't have a back-end, the html file names will be used as routes and these need to be explanatory, not cryptic for SEO purposes.

e.g. `.com/case-studies/science-museum-online collections`

Reply:
Proposing: /case-studies/ellen-macarthur-foundation, /case-studies/inclusive-classrooms and /case-studies/science-museum-online for now?

Reply:
We should probably have a convention:

case-studies/client-name/project-name

e.g.
/case-studies/ellen-macarthur-foundation/ce100
/case-studies/inclusive-classrooms/[project-name]
case-studies/science-museum-group/online-collections

can't remember what exact project names are, but replace the end with the project name on the copy

Reply:
forgot to tag @Cleop